main_routers/workshop_router.py

```
@@ -0,0 +1,1699 @@
+            "error": "Steamworks API错误",
+            "message": "请确保Steam客户端已运行且已登录"
+        }, status_code=503)
+    except Exception as e:
+        logger.error(f"发布到创意工坊失败: {e}")
+        return JSONResponse(content={"success": False, "error": str(e)}, status_code=500)
+
+def _publish_workshop_item(steamworks, title, description, content_folder, preview_image, visibility, tags, change_note):
+    """
+    在单独的线程中执行Steam创意工坊发布操作
+    """
+    # 在函数内部添加导入语句，确保枚举在函数作用域内可用
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)